src/api/l_physics_world.c

```diff
@@ -110,6 +110,44 @@ static int l_lovrWorldNewSphereCollider(lua_State* L) {
   return 1;
 }
 
+static int l_lovrWorldNewMeshCollider(lua_State* L) {
+  World* world = luax_checktype(L, 1, World);
+  lovrAssert(lua_istable(L, 2), "Vertices must be a table");
+  lovrAssert(lua_istable(L, 3), "Indices must be a table");
+  int vertexCount = luax_len(L, 2);
+  int indexCount  = luax_len(L, 3);
+  // TODO: this never gets deallocated
```

Comment:
Was there anything challenging about this?  Does ODE need the vertex/index arrays to stay allocated during the lifetime of the trimesh?

Reply:
ODE crashes if I those arrays are deallocated while rigid body is still alive. ODE also won't free those allocations for us.

Memory should be freed in `dGeomTriMeshDataDestroy()`. To know pointer values at that time there are two options:

1. Make `MeshShape` struct special and include pointers there. Currently all shapes use same struct so this is a bit awkward.
2. Add a separate map structure that would use shape->id as key and pointers to vertices and indices as value. Still need to figure out how to track this per-physics-world, as different worlds can have same id for different collider.
3. You also mentioned the need for `luax_readtriangles`. Not sure what memory ownership model would be best there.

If you have peference for how to solve this, I'd love to fix this loose end.